js/public/bindings.js

```diff
@@ -1,37 +1,40 @@
+'use strict'
 const bindings = {
     axes: {},
     buttons: {}
 }
-const setBinding = (e, values) => {
-    let binding = e
-        ? e.target.closest('div.binding')
-        : addBinding(values)
+const setBinding = (e, values = undefined) => {
```

Comment:
Defaulting to undefined not necessary / useful here (probs some typescript thingy ?)